### PR TITLE
Eliminate shadow declarations

### PR DESCRIPTION
**Pull request by bzsolt**
IoT.js-DCO-1.0-Signed-off-by: Zsolt Borbély zsborbely.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by yichoi** [comment]
LGTM. Good for readability. Thanks

File: src/platform/iotjs_module_i2c-linux-general.inl.h [hunk 1]
```diff
@@ -89,7 +89,7 @@ typedef struct I2cSmbusIoctlDataStruct {
 } I2cSmbusIoctlData;
 
 
-int fd;
+int current_fd;
 uint8_t addr;
```
Comment (hs0225) [inline src/platform/iotjs_module_i2c-linux-general.inl.h]: This global variables need to be declared as static.

Comment (bzsolt) [inline src/platform/iotjs_module_i2c-linux-general.inl.h]: I've updated this part.

**Comment by chokobole** [comment]
LGTM

**Pull request merged**